app/models/searchgov_crawler.rb

```diff
@@ -21,6 +21,8 @@ def initialize(domain:, skip_query_strings: true, delay: nil, srsly: false)
   end
 
   def crawl
+    Rails.logger.warn(disallowed_warning) unless @robotex.allowed? base_url
+
```

Comment:
What if for some reason @robotex is null?

Reply:
`@robotex` is initialized as part of the crawler's `initialize` method. It is never nil.